src/index.mli

```diff
@@ -169,6 +169,10 @@ module type S = sig
   (** [replace t k v] binds [k] to [v] in [t], replacing any existing binding of
       [k]. *)
 
+  val filter : t -> (key * value -> bool) -> unit
+  (** [filter t p] triggers a blocking merge where only bindings that satisfy
+      [p] are kept. *)
```

Comment:
This is an odd description to be user-facing: can we drop the reference to 'merge'?